spec/models/like_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  describe 'Validation tests for Likes' do
+    subject { Comment.new(author_id: 2, post_id: 2) }
+
+    before { subject.save }
+
+    it 'Find author_id to be an integer' do
+      subject.author_id = 'caro'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find author_id to be an integer greater or equal to zero' do
+      subject.author_id = -2
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find post_id to be an integer' do
+      subject.post_id = 'caro'
+      expect(subject).to_not be_valid
+    end
+
+    it 'Find post_id to be an integer greater or equal to zero' do
+      subject.post_id = -2
+      expect(subject).to_not be_valid
+    end
+  end
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job following the previous reviewer's advice of writing unit tests for Likes. 🚀  However, I'm sorry that he didn't point out more details about what should be tested. in this case, you tested that a `like` should have an author and should be associated with a post but, there's a method `#likes_counter` in your model that is not tested, and, according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/228967641-fa4d9b31-2872-420f-9ae6-4b4ae08fc450.png)

I strongly suggest you add a test to it! 🤓 🔧